apps/src/standaloneVideo/VideoJS.tsx

```diff
@@ -0,0 +1,58 @@
+import React, {useRef} from 'react';
+import videojs, {VideoJsPlayer, VideoJsPlayerOptions} from 'video.js';
+import 'video.js/dist/video-js.css';
+
+// Adapted from https://videojs.com/guides/react/
+
+export const VideoJS = (props: {options: VideoJsPlayerOptions}) => {
+  const videoRef: React.MutableRefObject<HTMLDivElement | null> = useRef(null);
+  const videoRefCallback = (ref: HTMLDivElement) => {
```

Comment:
This looks slightly different than the videoJS example, and I think is not necessary per React docs here? Am I missing something?

https://react.dev/reference/react/useRef#manipulating-the-dom-with-a-ref

Reply:
Good call.  Simplified.